libixx/src/index.rs

```diff
@@ -157,7 +157,15 @@ impl Index {
         .options
         .iter()
         .enumerate()
-        .find(|(idx, OptionEntry { labels: option, .. })| do_labels_match(*idx, option, &labels))
+        .find(
+          |(
+            idx,
+            OptionEntry {
+              scope_id: option_scope_id,
```

Comment:
Are we now searching through all scopes at once or one scope at the time?

Reply:
This is not the search function. This functions determinate the index of the option in the json metadata. Because option names are now only unique within a scope, the scope id must be included in the parameters of this query function to unique identify the option.